src/System.Collections.Immutable/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard;
```

Comment:
Maybe I missed something but I thought there were issues with things going in and out of the netcoreapp bubble (perhaps only for refs?) so I thought we were going to make a netcoreapp specific configuration for these.